src/renderer/shared/util/sendAsset.js

```diff
@@ -16,17 +14,38 @@ export default async function sendAsset({ net, asset, amount, receiver, address,
     throw new Error(`Invalid amount: "${amount}"`);
   }
 
-  const selectedAsset = ASSETS[asset];
-  const intents = await api.makeIntent({ [selectedAsset]: amount }, receiver);
-
-  const config = {
-    net,
-    address,
-    privateKey: new wallet.Account(wif).privateKey,
-    intents
-  };
-
-  const { response: { result, txid } } = await api.sendAsset(config, api.neoscan);
+  let result;
+  let txid;
+  if (keys(ASSETS).includes(asset)) {
+    const selectedAsset = ASSETS[asset];
+    const intents = await api.makeIntent({ [selectedAsset]: amount }, receiver);
+
+    const config = {
+      net,
+      address,
+      privateKey: new wallet.Account(wif).privateKey,
+      intents
+    };
+
+    ({ response: { result, txid } } = await api.sendAsset(config, api.neoscan));
+  } else {
+    ({ response: { result, txid } } = await Neon.doInvoke({
+      net,
+      address,
+      script: createScript(asset, 'transfer', [
+        address,
+        receiver,
+        new u.Fixed8(amount)
+      ], true),
+      privateKey: wif,
+      gas: 0,
+      intents: [{
```

Comment:
This drop of GAS shouldn't be necessary for invoking transfers anymore.  This was only needed by older versions of neon-js.

Reply:
Ok, I didn't know that.